### ISSUE TITLE
Notifications (aria-live) are not announced on click or tap

### ISSUE THREAD

**Issue by wisniewski94** [nolimits4web/Swiper#8016]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

https://swiperjs.com/demos/020-navigation/core

### Bug description

Swiper notifications are announced only if buttons are clicked with the keyboard, but not on tap/click.

### Expected Behavior

State of carousel is announced via aria-live (`.swiper-notification`) on TalkBack

### Actual Behavior

Tapping on navigation buttons is "silent"

### Swiper version

11.1.4

### Platform/Target and Browser Versions

macOS Chrome 136.0.7103.114

### Validations

- [x] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [x] Read the [docs](https://swiperjs.com/swiper-api).
- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [x] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [x] I'm willing to open a PR
